packages/AutoTDD-Core.package/ATDDNullTestRunner.class/instance/active..st

```diff
@@ -0,0 +1,2 @@
+accessing
+active: aBoolean
```

Comment:
This setter has no functionality? Is this correct?

Reply:
Yeah, NullObjects shouldn't provide any functionality

Reply:
Free coverage hellyeah :D